### PR TITLE
Portfolio: Desktop version

### PR DESCRIPTION
**Pull request by Masuduzzaman811**
## Desktop version of the portfolio

In this desktop branch, I have rescaled the project and changed the layout ( according to the Figma template ) for the desktop version.

### Key Requirements:

- Use media queries in the CSS file 
- Use Grid and FlexBox in the layout
- Use one input for the full name in the form

I've made the changes as per the requirements.  I've checked my output in the: 
- Chrome Browser 
- FireFox Browser 
- Microsoft Edge Browser

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -128,74 +149,190 @@ <h2 class="card-head">Multi-Post <br>Stories</h2>
...
+                <li class="about-li">
+                    <h3>Frameworks</h3>
+
+                    <nav class="about-icon">
+                        <img src="images/arrow-right.svg" alt="about_icon">
+                    </nav>
+                </li>
+                <li class="about-li">
+                    <h3>Skills</h3>
+
```
Comment (roshan-bajgain) [inline index.html:326]: - [x] Kindly use the same color as the Figma design for your `h3` tag.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- Nice PR description and title :+1: 
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:

## Required Changes ♻️
- [x] It would be better if you add `live demo` in your README file so that any user can get access to your page easily.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Masuduzzaman811) [reply index.html:326]: Done: Changed the color to `color: #344563;`

Comment (micheaol) [reply index.html:326]: [OPTIONAL]: Please ensure that you replicate the design as much as possible (e.g., font, colors, images, tests, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
image  | image

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @Masuduzzaman811 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by henrykgggg — COMMENTED** [review]
all looks perfect

**Comment by henrykgggg** [comment]
You have  make a great job so far, perfect only that you have to remove comments in the code

**Pull request merged**